charts/common/aws-secrets-manager/rds-only/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "v3.3.5"
```

Comment:
why 3.3.5?

I think we can stick to 1.6.1 for this as well

Reply:
https://github.com/awslabs/kubeflow-manifests/blob/54834fd03d0e241c7908b4a5e2d22e9a15267dbb/awsconfigs/common/aws-secrets-manager/base/deployment.yaml#L19
I got the version from the tag here:
https://gallery.ecr.aws/xray/aws-xray-daemon

Reply:
I see, thats a dummy image we use for this component, its used to just create a pod for mounting purposes. we can do 1.0.0 here